src/test/java/org/jscep/message/PkiMessageEncoderTest.java

```diff
@@ -156,6 +156,59 @@ public void simpleTestTripleDES() throws Exception {
 
         assertEquals(message, actual);
     }
+    
+    @Test
+    public void simpleTestAES192() throws Exception {
```

Comment:
Added a method called `encodeAndDecodeEnvelope(String cipher)` is called from the test method.